### PR TITLE
Apple Pay Interface Fix

### PR DESCRIPTION
**Pull request by mnicpt**
Getting undefined on `event` when passing as `e.<data>`.  Pulling out and setting into constant fixes this.

### CODE REVIEW

**Review by amyegan — APPROVED** [review]
lgtm

File: src/zoid/buttons/util.js [hunk 1]
```diff
@@ -89,19 +89,23 @@ export function applePaySession() : ?ApplePaySessionConfigRequest {
             };
 
             session.onpaymentmethodselected = (e) => {
```
Comment (gregjopa) [inline src/zoid/buttons/util.js:91]: Have you tried destructuring like this? Seems strange that the previous solution didn't work. Wondering if we need a fasly check on the args of these callbacks.
```js
session.onpaymentmethodselected = ({ paymentMethod }) => {
  listeners.paymentmethodselected({ paymentMethod });
};

Comment (mnicpt) [reply src/zoid/buttons/util.js]: That works!  Thanks @gregjopa

File: src/funding/applepay/config.jsx [hunk 2]
```diff
@@ -38,6 +38,8 @@ export function getApplePayConfig() : FundingSourceConfig {
         logoColors:  {
             [ BUTTON_COLOR.BLACK ]:  LOGO_COLOR.WHITE,
             [ BUTTON_COLOR.WHITE ]:  LOGO_COLOR.BLACK
-        }
+        },
+
+        shippingChange: true
```
Comment (gregjopa) [inline src/funding/applepay/config.jsx:43]: That's cool that ApplePay supports the shippingCallback! 💯

**Review by gregjopa — APPROVED** [review]

File: src/funding/applepay/config.jsx (hunk 2 above)
Comment (mnicpt) [reply src/funding/applepay/config.jsx:43]: We patch in off of Apple's Event listeners.  Now just have to get it working by upgrading LSAT.

**Pull request merged**